hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/schemas/V0490FileSchema.java

```diff
@@ -209,7 +206,7 @@ public static void dispatchSynthFileUpdate(
 
     public Bytes nodeStoreNodeDetails(@NonNull final ReadableNodeStore nodeStore) {
         final var nodeDetails = new ArrayList<NodeAddress>();
-        StreamSupport.stream(Spliterators.spliterator(nodeStore.keys(), nodeStore.sizeOfState(), DISTINCT), false)
+        nodeStore.keys().stream()
```

Comment:
Why does this usage of `.keys()` not need to be removed?

Reply:
I changed the `keys()` implementation to return a list of ids and not using the `KVState.keys()` implementation. I can change the name to `keysList` or something to not be confusing

Reply:
Ahh right, it's using the _store_ `.keys()` method, which doesn't use `KVState.keys()`. Got it, thanks!